packages/tools/client-debugger/client-debugger-view/src/components/ContainerDevtoolsView.tsx

```diff
@@ -0,0 +1,261 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IOverflowSetItemProps, IconButton, Link, OverflowSet, Stack } from "@fluentui/react";
+import {
+	ContainerDevtoolsFeature,
+	ContainerDevtoolsFeatureFlags,
+	ContainerDevtoolsFeatures,
+	GetContainerDevtoolsFeatures,
+	HasContainerId,
+	ISourcedDevtoolsMessage,
+	InboundHandlers,
+	handleIncomingMessage,
+} from "@fluid-tools/client-debugger";
+import React from "react";
+
+import { initializeFluentUiIcons } from "../InitializeIcons";
+import { useMessageRelay } from "../MessageRelayContext";
+import { AudienceView } from "./AudienceView";
+import { ContainerHistoryView } from "./ContainerHistoryView";
+import { ContainerSummaryView } from "./ContainerSummaryView";
+import { DataObjectsView } from "./DataObjectsView";
+import { Waiting } from "./Waiting";
+
+// TODOs:
+// - Allow consumers to specify additional tabs / views for list of inner app view options.
+// - History of client ID changes
+
+// Ensure FluentUI icons are initialized for use below.
+initializeFluentUiIcons();
```

Comment:
Unrelated to the PR but before I forget. Should we move this to the RootView so several other ones don't have to call it?

Reply:
Possibly, so long as our package exports only export component(s) in modules that contain this call. Though removing this will mean that test modules will have to call it instead. Probably still an improvement, but there are downsides.